src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java

```diff
@@ -347,25 +350,23 @@ public void testUpdateFeedbackQuestion() throws Exception {
     }
 
     private FeedbackQuestionAttributes getNewFeedbackQuestionAttributes() {
-        FeedbackQuestionAttributes fqa = new FeedbackQuestionAttributes();
-
-        fqa.courseId = "testCourse";
-        fqa.creatorEmail = "instructor@email.com";
-        fqa.feedbackSessionName = "testFeedbackSession";
-        fqa.giverType = FeedbackParticipantType.INSTRUCTORS;
-        fqa.recipientType = FeedbackParticipantType.SELF;
-        fqa.numberOfEntitiesToGiveFeedbackTo = 1;
-        fqa.questionNumber = 1;
-
         FeedbackTextQuestionDetails questionDetails = new FeedbackTextQuestionDetails("Question text.");
-        fqa.questionType = FeedbackQuestionType.TEXT;
-        fqa.setQuestionDetails(questionDetails);
-
-        fqa.showGiverNameTo = new ArrayList<>();
-        fqa.showRecipientNameTo = new ArrayList<>();
-        fqa.showResponsesTo = new ArrayList<>();
 
-        return fqa;
+        return FeedbackQuestionAttributes.builder()
+                .withCourseId("testCourse")
+                .withCreatorEmail("instructor@email.com")
+                .withFeedbackSessionName("testFeedbackSession")
+                .withGiverType(FeedbackParticipantType.INSTRUCTORS)
+                .withRecipientType(FeedbackParticipantType.SELF)
+                .withNumOfEntitiesToGiveFeedbackTo(1)
+                .withQuestionNumber(1)
+                .withQuestionType(FeedbackQuestionType.TEXT)
+                .withQuestionMetaData(new Text(JsonUtils.toJson(questionDetails,
+                        FeedbackQuestionType.TEXT.getQuestionDetailsClass())))
```

Comment:
Any particular reason for using this as the question metadata?

Reply:
It's the same as the method below without making a call to it (basically setting the question details). Since I'm using a builder, I don't want to make a call to the method. 
https://github.com/TEAMMATES/teammates/blob/afa4dfe6170e32beb64be5d37ac627fffeee6b3c/src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java#L549-L554

Reply:
Fixed this by creating an overloaded `withQuestionMetaData()`

```java
        public Builder withQuestionMetaData(Text questionMetaData) {
            if (questionMetaData != null) {
                feedbackQuestionAttributes.questionMetaData = questionMetaData;
            }
            return this;
        }

        public Builder withQuestionMetaData(FeedbackQuestionDetails questionDetails) {
            if (questionDetails != null) {
                feedbackQuestionAttributes.setQuestionDetails(questionDetails);
            }
            return this;
        }
```